patterns/1-initial/creating_an_innersource_strategy.md

```diff
@@ -0,0 +1,71 @@
+## Title
+
+Creating an InnerSource Strategy
+
+## Patlet / Summary
+
+Sometimes, it is difficult to convince people of the relevance of InnerSource for your organization and/or to get support from management. Creating an InnerSource strategy, that connects your InnerSource approach and activities to the goals and the overall strategy of your organization, can help in this regard.
+
+## Problem
+
+- Despite an organization has an InnerSource program, it is challenging to establish InnerSource in that organization
+- Missing management support and low awareness in some or many development teams are among the reasons
+- Potentially, the InnerSource program itself is missing clear goals and/or approaches how to achieve those.
+
+## Context
+
+- InnerSource is not an end in itself. It must deliver clear benefits to your organization and support its goals
+- If InnerSource and its benefits is mainly explained and positioned in a rather abstract way, it might be difficult to convince management and a critical mass of the development teams
+- People see InnerSource as a rather abstract methodology which does not really fit to their situation. They might say/think "InnerSource sounds great, but it is not relevant for our organization"
+- People struggle to make a connection between InnerSource and their goals/the goals of their organisation
+
+## Forces
+
+TODO
```

Comment:
Porting these potential forces here from the current "Problem" section.

Are these good "constraints that can be changed at a cost"? Is the solution that you are proposing "changing one or more of these forces in order to solve the problem"?

I am often confused myself between what goes into the **Context** and **Forces** section, party because these ideas from a rather scientific context. So I cannot blame you if you are confused here as well :)


```suggestion
- The InnerSource program itself is missing clear goals and/or approaches how to achieve those
- Missing management support for the InnerSource program
- Low awareness of the InnerSource program in some development teams
```